color/src/colorspace.rs

```diff
@@ -525,14 +525,19 @@ impl ColorSpace for Rec2020 {
     }
 }
 
+#[expect(
+    clippy::doc_markdown,
+    reason = "these color space names should not be surrounded in backticks"
+)]
```

Comment:
Instead, you can add a `.clippy.toml` in the root of this repository with this line in it:

```
doc-valid-idents = ["ACEScc", "ACEScct", ".."]
```

Not sure if you'll need something for ACES2065-1.